edit: shift 2 if +line(:column) is given

listentolist opened the pull request:
This PR fixes an issue, that +line(:column) is not recognized whith edit command. For example: Running `edit $(it) +10:2` in a connected terminal opens the current buffile and the new file "+10:2".

listentolist committed:
edit: shift 2 if +line(:column) is given

alexherbo2 commented:
Why `+<line>[:<column>]` is not recognized in the first place? 🤔

alexherbo2 commented:
The problem was because I used `$2` instead of `$1`. 🤦

listentolist commented:
That fixes it too. Just out of curiosity: Was there a problem with the solution, that I used?

Btw: Thank you for this plugin! I am just discovering the power of it...

alexherbo2 commented:
I have been undoing my changes.  Could you add a copyright waiver before the first commit, so I can merge?

listentolist committed:
Tilman List Copyright Waiver

I dedicate any and all copyright interest in this software to the
public domain.  I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors.  I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.
